nekoyume/Assets/_Scripts/BlockChain/ActionHandler.cs

```diff
@@ -214,6 +229,17 @@ protected async UniTask UpdateCurrentAvatarStateAsync()
             }
         }
 
+        protected static void UpdateHackAndSlashBuffState<T>(
+            ActionBase.ActionEvaluation<T> evaluation) where T : ActionBase
+        {
+            var state = GetHackAndSlashBuffState(evaluation);
+
+            if (state is { })
+            {
+                States.Instance.SetHackAndSlashBuffState(state);
+            }
+        }
+
```

Comment:
If the `GetHackAndSlashBuffState()` function will be async function,

```suggestion
        protected static UniTask UpdateHackAndSlashBuffStateAsync<T>(
            ActionBase.ActionEvaluation<T> evaluation) where T : ActionBase
        {
            var state = await GetHackAndSlashBuffStateAsync(evaluation);

            if (state is { })
            {
                States.Instance.SetHackAndSlashBuffState(state);
            }
        }

```